### ISSUE TITLE
v5.0.2 - cannot find module '[...]/node_modules/use-debounce/dist/index.js'

### ISSUE THREAD

**Issue by icflorescu** [xnimorz/use-debounce#84]
I'm getting this error in a TS + Next.js application after upgrading `use-debounce` to `5.0.2`.
Things worked just fine prior to (and including) `5.0.1`, so I'm suspecting it has something to do with these new lines that were introduced in `package.json` in one of the last commits:
```
"exports": {
    ".": {
      "import": "./dist/index.module.js",
      "require": "./dist/index.js"
    }
  }
```
...and the fact that there's no `dist` sub-folder created when installing the package; just `lib` and `esm`. 
Removing the lines solved the problem for me.

**Comment by icflorescu** [comment]
Please do let me know if there's anything more I can do to help with this issue.

**Comment by xnimorz** [comment]
Hi @icflorescu 
Thank you a lot!
I've just fixed this, published the 5.0.3 version, please, try it, the issue should be solved

**Comment by icflorescu** [comment]
Wow, that was fast :-)
Fixing the paths in `5.0.3` solved the issue, of course.
Thanks a lot and keep up the good work!
